jetbrains-core/src/software/aws/toolkits/jetbrains/ui/ResourceSelector.kt

```diff
@@ -138,6 +138,12 @@ class ResourceSelector<T> : ComboBox<T>() {
         }
     }
 
+    fun values(): List<T> = ArrayList<T>().apply {
+        for (i in 0..(itemCount - 1)) {
```

Comment:
prefer the functional style here: `fun values(): List<T> = (0 until itemCount).map { getItemAt(it) }`

Reply:
Done. Returned a set from here.